test/framework/flux.go

```diff
@@ -247,6 +260,123 @@ func (e *ClusterE2ETest) initGit(ctx context.Context) {
 	e.GitClient = g.Client
 }
 
+func (e *ClusterE2ETest) workloadClusterConfGitPath(w *WorkloadCluster) string {
+	return e.clusterConfGitPathFromName(w.ClusterName)
+}
+
+func (e *ClusterE2ETest) workloadClusterConfigGitPath(w *WorkloadCluster) string {
+	return filepath.Dir(filepath.Join(e.GitWriter.Dir(), e.workloadClusterConfGitPath(w)))
+}
+
+func (e *ClusterE2ETest) buildWorkloadClusterConfigFileForGit(w *WorkloadCluster) {
+	b := w.generateClusterConfigYaml()
+	g := e.GitWriter
+	p := e.workloadClusterConfigGitPath(w)
+
+	if _, err := os.Stat(p); errors.Is(err, os.ErrNotExist) {
+		err := os.MkdirAll(p, os.ModePerm)
+		if err != nil {
+			w.T.Errorf("creating directory [%s]: %v", g.Dir(), err)
+		}
+	}
+
+	_, err := g.Write(e.workloadClusterConfGitPath(w), b, filewriter.PersistentFile)
+	if err != nil {
+		w.T.Errorf("Error writing cluster config file to local git folder: %v", err)
```

Comment:
nit:
```suggestion
		w.T.Errorf("error writing cluster config file to local git folder: %v", err)
```

Reply:
Since this is logging to the console, would it not make sense to have it capitalized like all the other logs?